services/src/main/java/org/jboss/windup/web/services/service/AnalysisContextService.java

```diff
@@ -129,7 +129,10 @@ public AnalysisContext update(Long analysisContextId, AnalysisContext analysisCo
         this.loadPackagesToAnalysisContext(analysisContext);
         analysisContext.setMigrationProject(original.getMigrationProject());
 
-        return entityManager.merge(analysisContext);
+        analysisContext.getMigrationProject().setProvisional(Boolean.FALSE);
```

Comment:
I agree that this is the best place for now, but could you please add a comment?